Makefile

```diff
@@ -122,11 +122,19 @@ clean: clean_docs
        
 
 .PHONY: integration ## Run integration tests
-integration: GODOG_OPTS = --godog.tags=$(GOOS)
 integration:
-	@$(call check_defined, BUNDLE_LOCATION, "'make integration' requires BUNDLE_LOCATION to contain the full path to a bundle file")
-	@$(call check_defined, PULL_SECRET_FILE, "'make integration' requires PULL_SECRET_FILE to point to a file with the pull secret to use")
-	@go test --timeout=90m $(REPOPATH)/test/integration -v --tags=integration $(GODOG_OPTS) $(BUNDLE_LOCATION) $(PULL_SECRET_FILE)
+GODOG_OPTS = --godog.tags=$(GOOS)
+ifndef PULL_SECRET_FILE
+	PULL_SECRET_FILE = --pull-secret-file=$(HOME)/Downloads/crc-pull-secret
```

Comment:
@jsliacan why we want to hard code this pull secret file instead using the ENV variable?

Reply:
@praveenkumar I'm not sure I understand, it's only hard coded if ENV variable is not defined. So in practice, it stays as before, except now it also has a default value.